lib/msal-common/test/cache/entities/CredentialEntity.spec.ts

```diff
@@ -112,6 +112,48 @@ describe("CredentialEntity.ts Unit Tests", () => {
             expect(CredentialEntity.getCredentialType(atKey)).toEqual(CredentialType.ACCESS_TOKEN);
         });
 
+        it("Host name with a single long TLD - Is access token", () => {
```

Comment:
Minor nit: Can we make sure there's a comment or something that indicates "TLD" stands for top-level domain somewhere? I think readability here is crucial since RegEx  are notoriously hard to maintain.

Reply:
Done

Reply:
Thanks!